package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/rdfjs-base/serializer-rdfjs",
   "dependencies": {
     "@rdfjs/sink": "^2.0.0",
+    "@rdfjs/types": "^1.1.0",
```

Comment:
```suggestion
```

The test files are not executed. They are only used for string comparison as the expected generated output. As the package doesn't have any type definitions at the moment, that package should not be required.